### PR TITLE
Design/initial database

### LINKED ISSUE #5: Initial database design

### PR DESCRIPTION
**Pull request by YnkDK**
**What type of PR is this?**
design

**What this PR does / why we need it**:
Adds the initial models for the database

**Which issue(s) this PR fixes**:

Fixes #5 

**Special notes for your reviewer**:
I have read that Docker is easy to add later on, so that is why this is not included yet. 

All attributes in `User` are named according to standard claims in OpenID Connect

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: l10n/DataAccess/Models/MetaDataEntity.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        /// The date and time for when the user was first created
+        /// </summary>
+        [Required]
+        public DateTime Created { get; set; }
+
+        /// <summary>
+        /// Automatically set on insert or update.
+        /// The property is also treated as a concurrency token. This ensures that an exception is thrown if anyone else has modified this row since the row was queried.
+        /// </summary>
+        [Timestamp]
```
Comment (YnkDK) [inline l10n/DataAccess/Models/MetaDataEntity.cs:18]: This is to use optimistic concurrency control. Related: https://docs.microsoft.com/en-us/ef/core/modeling/concurrency#timestamprow-version

Comment (odinstown76) [reply l10n/DataAccess/Models/MetaDataEntity.cs:18]: Nice way of handling it

File: l10n/DataAccess/Models/Phrase.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        [MaxLength(2047)]
+        [Required]
+        public string Key { get; set; }
+
+        /// <summary>
+        /// The SHA256 of the Key attribute
+        /// </summary>
+        [MaxLength(32)]
+        [Required]
+        public byte[] KeyHash { get; set; }
```
Comment (odinstown76) [inline l10n/DataAccess/Models/Phrase.cs]: What role does the KeyHash play?

File: l10n/DataAccess/Models/Phrase.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+using System.ComponentModel.DataAnnotations;
+
+namespace DataAccess.Models
+{
+    public class Phrase : OrganizationBoundEntity
+    {
+        /// <summary>
+        /// Primary key
+        /// </summary>
+        public Guid PhraseId { get; set; }
```
Comment (odinstown76) [inline l10n/DataAccess/Models/Phrase.cs:11]: What is the reasoning behind using Guid and not a sequential uint?

**Review by odinstown76 — COMMENTED** [review]
Just need some clarification on my comments :)

**Review by odinstown76 — APPROVED** [review]

**Pull request merged**